front/src/reducers/osrdconf/stdcmConf/index.ts

```diff
@@ -276,7 +276,7 @@ export const stdcmConfSlice = createSlice({
       }
       const newPathSteps = state.stdcmPathSteps.map((step) =>
         step.id === action.payload.pathStepId
-          ? ({ ...step, ...newPathStep } as StdcmPathStep)
+          ? ({ ...step, ...newPathStep.location } as StdcmPathStep)
```

Comment:
You can probably remove this
```suggestion
          ? ({ ...step, ...newPathStep.location })
```